app/Http/Requests/Manager/Notification/BulkUpdateStatusRequest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Http\Requests\Manager\Notification;
+
+use App\Enums\NotificationStatus;
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rules\Enum;
+
+class BulkUpdateStatusRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     *バリデーションルール
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'notifications' => ['required', 'array', 'min:1'],
+            'notifications.*' => ['required', 'integer', 'exists:notifications,id,deleted_at,NULL'],
+            'status' => ['required', 'string', new Enum(NotificationStatus::class)],
```

Comment:
Slackで共有したドキュメントに記載されている次のような記述にしなかったのには、何か理由がありましたでしょうか？
https://laravel.com/docs/11.x/validation#rule-enum
こちらは私が想定できなかったケースがあったかもしれないので、確認でした！

```suggestion
            'status' => ['required', 'string', Rule::enum(StatusEnum::class)],
```

Reply:
確認見落としておりました。
以後、気を付けます！